docs/services/navigation/_index.md

```diff
@@ -135,15 +144,14 @@ Go to your robot's **Code Sample** tab on the [Viam app](https://app.viam.com) f
 
 ### Mode
 
-{{% alert title="Stability Notice" color="note" %}}
-
-The `manual` mode is experimental and will have no effect.
+Get the `Mode` the service is operating in.
 
-{{% /alert %}}
+There are three options for modes: `MODE_UNSPECIFIED`, `MODE_MANUAL`, or `MODE_WAYPOINT`.
 
-Get the `Mode` the service is operating in.
-There are two options for modes: `manual` and `waypoint`.
-If you set the mode to `waypoint`, the service starts looking for added waypoints, effectively beginning navigation.
+- `MODE_UNSPECIFIED`: Do not perform autonomous navigation.
```

Comment:
The default is MODE_MANUAL. MODE_UNSPECIFIED is just in proto, since all enums have to have an unspecified option. We don't need to document MODE_UNSPECIFIED.

Reply:
Ok, that makes sense. Have addressed!